A36-xds-for-servers.md

```diff
@@ -0,0 +1,332 @@
+A36: xDS-Enabled Servers
+------------------------
+* Author(s): [Eric Anderson](https://github.com/ejona86), [Doug
+  Fawley](https://github.com/dfawley), [Mark Roth](https://github.com/markdroth)
+* Approver: markdroth
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2020-11-19
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Did you create the thread? You should do so ASAP if not since it needs to wait at least 2 weeks.

Reply:
Well, given that Go's section is apparently all wrong, I think we'd need to at least get it in working shape or remove it before then to avoid confusion.

Reply:
I pushed a commit which updates the Go section. PTAL @dfawley @ejona86